packages/language-server/README.md

```diff
@@ -0,0 +1,89 @@
+# `@mdx-js/language-service`
+
+## What is this?
+
+This package provides a [language server][lsp] for [MDX].
+The language server provides intellisense based on [TypeScript], as well as some markdown specific
+features.
+
+## When should I use this?
+
+You can use this package if you want to enhance your editor with intellisense for [MDX] files.
+Some editors can consume this package directly, others need a plugin in order to consume this package.
+
+## Install
+
+This package is not published yet.
+
+## Use
+
+Usage of this package depends on your editor integration.
+
+This package provides the CLI `mdx-language-server`.
+Because this is based on [`vscode-languageserver`][vscode-languageserver], the same transports are supported.
+
+## Language server features
+
+This language server supports the following features:
+
+- Markdown definitions
+- Markdown hover hints
+- TypeScript completions
+- TypeScript definitions
+- TypeScript diagnostics
+- TypeScript hover hints
+- TypeScript references
```

Comment:
Good idea to list the features!
Future enhancement: Potentially even better could be a recording of it. Or relevant links on what these features are.

Reply:
I like the idea of a small recording! Although I think it’s sufficient to add a recording to the VSCode extension readme instead. The language server doesn’t have a GUI on its own and editor experience may differ per client.

Last time I tried my screen recorder was broken though :sweat_smile: 

I don’t think it’s very useful to add links on what these features are. To me that feels like linking to the HTML specification to explain an end user what a website is: technical detail.